typescript/infra/package.json

```diff
@@ -11,19 +11,19 @@
     "@aws-sdk/client-s3": "^3.74.0",
     "@ethersproject/experimental": "^5.6.2",
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
-    "@types/mocha": "^9.1.0",
-    "@types/node": "^16.9.1",
-    "@types/yargs": "^17.0.10",
     "asn1.js": "5.4.1",
-    "chai": "^4.3.4",
     "dotenv": "^10.0.0",
     "prom-client": "^14.0.1",
     "yargs": "^17.4.1"
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.5",
     "@nomiclabs/hardhat-waffle": "^2.0.2",
+    "chai": "^4.3.4",
```

Comment:
upon running `yarn` locally it put this below the `"@types/yargs"`, may be why a CI step is failing